examples/clusters/userdata-aws-manager

```diff
@@ -20,6 +20,11 @@ AMPAGENT_VERSION=${APP_VERSION:-latest}
 SYSTEMD_DOCKER_OVERRIDE=/etc/systemd/system/docker.service.d/docker.conf
 SYSV_DOCKER_DEFAULT=/etc/default/docker
 
+export TAG=${APP_VERSION:-latest}
```

Comment:
not sure about setting the TAG variable to the tag of the ampagent image. It will break the way ampagent sets the environment for core stack files (when tag is not set, it uses the Version var, which is something like 0.18.0-dev on dev versions, 0.18.0 on released versions, and latest on dev builds for cloud deployments (make VERSION=latest clean-ampagent build-ampagent).